src/notification-hubs/prod/notification_hub_ntfns_common.tf

```
@@ -57,3 +57,61 @@ resource "azurerm_notification_hub_authorization_rule" "common_default_full" {
   send                  = true
   listen                = true
 }
+
+resource "azurerm_monitor_metric_alert" "alert_nh_common_pns_errors" {
+
+  name                = "[IOCOM|NHLegacy] Push Notification Service errors."
+  resource_group_name = azurerm_notification_hub_namespace.common.resource_group_name
+
+  scopes              = [azurerm_notification_hub_namespace.common.id]
+  description         = "Notification Hub Legacy incurred in PNS errors, please check. Runbook: not needed."
+  severity            = 1
+  window_size         = "PT30M"
```

Do you think it may be beneficiary to reduce the window size? As of now, the notification would be triggered if the threshold is reached during the whole 30 minutes period
```suggestion
  window_size         = "PT10M"
```

---

We want that the sum over the last 30 minutes is greater than 30k. This should trigger at least 30 times for a spike of push. I know that now it is pretty much, but we need to be kinda "spammed" from this alert.